apps/nextjs-website/src/editorialComponents/Feature/Subtitle.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable valid-typeof */
+import { Link, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+export interface SubtitleProps {
+  subtitle: string;
+  textLink?: string;
+  url?: string;
+  theme: 'dark' | 'light';
+}
+
+const Subtitle = ({ subtitle, textLink, url, theme }: SubtitleProps) => {
+  return (
+    <Stack spacing={3} justifyContent='center' alignItems='center'>
+      <Typography
+        color={theme === 'light' ? 'text.primary' : 'background.paper'}
+        variant='body2'
+        textAlign='center'
+      >
+        {subtitle}
+      </Typography>
+      {textLink !== undefined && (
```

Comment:
```suggestion
      {textLink && (
```
i think you can omit that